Broken link fixed week29 (#200)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/tools/kusto-explorer-using.md

> 1. Provide a parameter for every query parameter in the URI
> In this case:

Reviewer: ```suggestion
1. Provide a parameter for every query parameter in the URI, such as:
```
